apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/main/java/gov/cms/bfd/pipeline/bridge/model/Mcs.java

```diff
@@ -1,25 +1,44 @@
-  public static final String CARR_CLM_CNTL_NUM = "CARR_CLM_CNTL_NUM";
-  public static final String CLM_FROM_DT = "CLM_FROM_DT";
-  public static final String CLM_ID = "CLM_ID";
-  public static final String CLM_THRU_DT = "CLM_THRU_DT";
-  public static final String HCPCS_CD = "HCPCS_CD";
-  public static final String HCPCS_1ST_MDFR_CD = "HCPCS_1ST_MDFR_CD";
-  public static final String HCPCS_2ND_MDFR_CD = "HCPCS_2ND_MDFR_CD";
-  public static final String ICD_DGNS_CD = "ICD_DGNS_CD";
-  public static final String ICD_DGNS_VRSN_CD = "ICD_DGNS_VRSN_CD";
-  public static final String LINE_1ST_EXPNS_DT = "LINE_1ST_EXPNS_DT";
-  public static final String LINE_ICD_DGNS_CD = "LINE_ICD_DGNS_CD";
-  public static final String LINE_ICD_DGNS_VRSN_CD = "LINE_ICD_DGNS_VRSN_CD";
-  public static final String LINE_LAST_EXPNS_DT = "LINE_LAST_EXPNS_DT";
-  public static final String LINE_NUM = "LINE_NUM";
-  public static final String NCH_CARR_CLM_SBMTD_CHRG_AMT = "NCH_CARR_CLM_SBMTD_CHRG_AMT";
-  public static final String ORG_NPI_NUM = "ORG_NPI_NUM";
+  public static final String BENE_ID = CarrierClaimColumn.BENE_ID.name();
+  public static final String CARR_CLM_CNTL_NUM = CarrierClaimColumn.CARR_CLM_CNTL_NUM.name();
+  public static final String CLM_FROM_DT = CarrierClaimColumn.CLM_FROM_DT.name();
+  public static final String CLM_ID = CarrierClaimColumn.CLM_ID.name();
+  public static final String CLM_THRU_DT = CarrierClaimColumn.CLM_THRU_DT.name();
+  public static final String HCPCS_CD = CarrierClaimColumn.HCPCS_CD.name();
+  public static final String HCPCS_1ST_MDFR_CD = CarrierClaimColumn.HCPCS_1ST_MDFR_CD.name();
+  public static final String HCPCS_2ND_MDFR_CD = CarrierClaimColumn.HCPCS_2ND_MDFR_CD.name();
+  public static final List<String> ICD_DGNS_CD =
+      listNumberedEnumNames(CarrierClaimColumn.values(), CarrierClaimColumn.ICD_DGNS_CD1);
+  public static final List<String> ICD_DGNS_VRSN_CD =
+      listNumberedEnumNames(CarrierClaimColumn.values(), CarrierClaimColumn.ICD_DGNS_VRSN_CD1);
+  public static final String LINE_1ST_EXPNS_DT = CarrierClaimColumn.LINE_1ST_EXPNS_DT.name();
+  public static final String LINE_ICD_DGNS_CD = CarrierClaimColumn.LINE_ICD_DGNS_CD.name();
+  public static final String LINE_ICD_DGNS_VRSN_CD =
+      CarrierClaimColumn.LINE_ICD_DGNS_VRSN_CD.name();
+  public static final String LINE_LAST_EXPNS_DT = CarrierClaimColumn.LINE_LAST_EXPNS_DT.name();
+  public static final String LINE_NUM = CarrierClaimColumn.LINE_NUM.name();
+  public static final String NCH_CARR_CLM_SBMTD_CHRG_AMT =
+      CarrierClaimColumn.NCH_CARR_CLM_SBMTD_CHRG_AMT.name();
+  public static final String ORG_NPI_NUM = CarrierClaimColumn.ORG_NPI_NUM.name();
+
+  /**
+   * The maximum number of diagnosis column names available for {@link ICD_DGNS_CD} and {@link
+   * ICD_DGNS_VRSN_CD}. Used when looping over both sets of fields.
+   */
+  public static final int MAX_DIAGNOSIS_CODES =
+      Math.min(ICD_DGNS_CD.size(), ICD_DGNS_VRSN_CD.size());
```

Comment:
In FISS we used `Math.max` for both, should those use `min` as well?

Reply:
Thanks for catching that.  Yes those should have used `min` as well.  It's fixed now.